src/composables/ViewRoutingSlip/useTransactionDataTable.ts

```diff
@@ -0,0 +1,79 @@
+import { Invoice, InvoiceDto, LineItem, Reference } from '@/models/Invoice'
+import { ref, toRefs, watch } from '@vue/composition-api'
+
+import { createNamespacedHelpers } from 'vuex-composition-helpers'
+
+const routingSlipModule = createNamespacedHelpers('routingSlip') // specific module name
+const { useGetters } = routingSlipModule
+
+// Composable function to inject Props, options and values to TransactionDataTable component
+export default function useTransactionDataTable (props, _) {
```

Comment:
second argument is not needed if not using

Reply:
thanks, removed it